gobblin-runtime/src/main/java/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -133,6 +133,12 @@ public MRJobLauncher(Properties jobProps, Configuration conf) throws Exception {
     // Put job configuration properties into the Hadoop configuration so they are available in the mappers
     JobConfigurationUtils.putPropertiesIntoConfiguration(this.jobProps, this.conf);
 
+    // Turn on mapreduce output compression by default
```

Comment:
The MR job does not use a `FileOutputFormat` to write data, so this has no effect. You don't need to do this here.